### PR TITLE
locate build name and clarify release id in README

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by murbanoao** [comment]
@oaojesse take a look when you get a sec.

File: README.md [hunk 1]
```diff
@@ -102,11 +102,17 @@ the unittest portion of the automated build process”. This is run by integrate
 ### integrate
 
 Executes the CI pipeline for the most recent commit of the local repo (pull,
-make, unit test, push commit and build). After unit testing and repo integration
-the CI server (Circle CI?) will build and test the commit.
+make, unit test, push commit and build). After unit testing and Circle CI
```
Comment (oaojesse) [inline README.md]: extra 'and' at the end of the line?

**Comment by oaojesse** [comment]
Other than that it all looks good.

**Comment by murbanoao** [comment]
thanks @oaojesse! merging

**Pull request merged**